.gitignore

```diff
@@ -4,8 +4,8 @@ build/artifacts
 phpunit.xml
 phpunit.functional.xml
 test_services.json
-Config
-./Makefile
+/Config
```

Comment:
This line was meant to exclude a file at the project root named `Config` but was also excluding a folder from the AWS manifests for a service named `config`.